frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -404,12 +404,14 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
     {
       key: 'usedStorage',
       query:
-        'sum by (instance) ((max by (device, instance) (node_filesystem_size_bytes{device=~"/.*"})) - (max by (device, instance) (node_filesystem_free_bytes{device=~"/.*"})))',
+        'sum by (instance) ((max by (device, instance) (node_filesystem_size_bytes{device=~"/.*"})) - (max by (device, instance) (node_filesystem_free_bytes{device=~"/.*"}))) or ' +
+        'sum by (instance) ((max by (volume, instance) (windows_logical_disk_size_bytes)) - (max by (volume, instance) (windows_logical_disk_free_bytes)))',
```

Comment:
```suggestion
        'sum by (instance) ((max by (device, instance) (node_filesystem_size_bytes{device=~"/.*"})) - (max by (device, instance) (node_filesystem_free_bytes{device=~"/.*"}))) + ' +
        'sum by (instance) ((max by (volume, instance) (windows_logical_disk_size_bytes)) - (max by (volume, instance) (windows_logical_disk_free_bytes)))',
```

Assuming you want the total used storage from **both** windows and linux nodes, I believe this should add the two, since doing an OR will only `group_left` and return the left value for the same labelset.

Reply:
PS. I don't know what this query is for exactly in UI, I'm basing my understanding from the earlier discussion and the context around this code snippet (`usedStorage`), so feel free to correct me if I missed something here. :)

Reply:
@rexagod  The intent is to return what is available from these two queries. If it is executed for a Windows node only the windows part of the query will return a value. This query is used on the individual node page of each of the nodes present in the cluster to display usage graphs.

Reply:
Ah, in that case, for a per-node basis, `or` should be the best choice. 👍🏼

Reply:
I didn't notice this earlier but these are `node_exporter` metrics, hence these will always be exposed on a per-node basis, i.e., to be explicit, `or` should be the right choice here.